src/config.rs

```diff
@@ -711,6 +711,14 @@ impl Context {
         if key == Config::SentboxWatch {
             self.last_full_folder_scan.lock().await.take();
         }
+
+        if matches!(
+            key,
+            Config::Configured | Config::Displayname | Config::Selfavatar | Config::PrivateTag
```

Comment:
For `Configured` this won't work because `set_raw_config_bool()` is used to set it, also a test is needed. Probably `test_configuration()` catches some other event, need to debug

Reply:
I moved this test to python because there we can use the real configure method more easily (because python already has online tests with real accounts).